cub/device/dispatch/dispatch_radix_sort.cuh

```diff
@@ -1498,6 +1498,11 @@ struct DispatchRadixSort :
                     }
                 }
 
+                if (error != cudaSuccess)
```

Comment:
Everything that sets `error` before this place, also checks it, right? I fail to see why this check is needed.

Reply:
This check is needed to break out of the `for (int current_bit = ...)` loop in case of error.

Reply:
Ah, got it, thanks!